marketplace-info/marketplace-info_shacl.ttl

```diff
@@ -0,0 +1,135 @@
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+marketplace-info:MarketplaceInfoShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:node marketplace-info:DomainShape ;
+                  sh:order 0 ;
+                  sh:path marketplace-info:domain ],  
+                [  sh:minCount 1 ;
+                  sh:node marketplace-info:ContentDataShape ;
+                  sh:order 1 ;
+                  sh:path marketplace-info:contentData ],
+                [ sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:node marketplace-info:DataShape ;
+                  sh:name "data object" ;
+                  sh:description "data object with property for, id" ;
+                  sh:order 2 ;
+                   sh:path marketplace-info:data ],           
+                [ sh:node marketplace-info:BundleDataShape ;
+                  sh:order 3 ;
+                  sh:path marketplace-info:bundleData ] ;
+    sh:targetClass marketplace-info:MarketplaceInfo .
+
+marketplace-info:DomainShape a sh:NodeShape ;
+    sh:property  [ sh:datatype xsd:string ;
+                  sh:in ("hdmap" "environment-model" "ositrace" "scenario" "surface-model" "service" "other" ) ;
+                  sh:message "Validation of domain type failed!"@en ;
+                  sh:description "Choose domain type."@en ;
+                  sh:name "domain"@en ;
+                  sh:order 0 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:domain ];
+    sh:targetClass marketplace-info:Domain .
+
+marketplace-info:ContentDataShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:string ;
+                  sh:in ("Documentation" "Metadata" "Validation" "Visualization") ;
+                  sh:message "Validation of type failed!"@en ;
+                  sh:description "Choose type of link."@en ;
+                  sh:name "type"@en ;
+                  sh:order 1 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:dataType ],
+                [ skos:example "link to content data, screenshot, video, routing, 3d preview"@en ;
+                  sh:description "Reference to content data"@en ;
+                  sh:message "Validation of content failed!"@en ;
+                  sh:name "link"@en ;
+                  sh:node marketplace-info:LinkShape ;
+                  sh:order 2 ;
+                  sh:maxCount 1 ;
+                  sh:minCount 1 ;
+                  sh:path marketplace-info:contentData ] ;
+    sh:targetClass marketplace-info:ContentData .
+
+
+
+marketplace-info:DataShape a sh:NodeShape ;
```

Comment:
Wollen wir diese Shape dann nicht in `ContractShape` umbenennen? 
Das einzige Attribut ist `contractId`

Reply:
done